### ISSUE TITLE
Some input broken with Alpha 75

### ISSUE THREAD

**Issue by bes-internal** [Eugeny/terminus#787]
Version: 1.0.75
Platform: win32 10.0.17763
Plugins: none

1. `Home`  and `End` hotkeys do not work in linux console (ssh). Both xterm/hterm. If reasign these hostkeys then home and end buttons work.
2. Sometimes copy and paste by third mouse button insert 0~<copied text>1~ with terminus alert "Illegal sequence"

**Comment by jbelien** [comment]
I have the exact same issue.

```
Version: 1.0.77
Platform: win32 10.0.17134
Plugins: none
```

**Comment by sburkett** [comment]
I have a similar problem with Home/End specifically. I removed them and remapped them, but it didn't help. Running 1.0.81 alpha. Interesting enough, SHIFT-END works ...

**Comment by sburkett** [comment]
Also noticing that right click to paste isn't working now either.

**Comment by Eugeny** [comment]
@sburkett is right click set up to paste in Settings > Terminal?

**Comment by sburkett** [comment]
Yes. It sometimes works, but only once, then I can't paste the same thing again. Also, if I copy something, then switch over to another tab, and paste, its the same thing. As long as I don't type anything else in that tab, I can paste, but the minute I do anything else in that tab, I can't
